snipeit.sh

```diff
@@ -216,34 +221,36 @@ case $distro in
 
 		webdir=/var/www
 		echo -e "\n* Updating Debian packages in the background... ${spin[0]}\n"
-		apt-get update >> /var/log/snipeit-install.log & pid=$! 2>&1
+		log "apt-get update" & pid=$!
 		wait
-		apt-get upgrade >> /var/log/snipeit-install.log & pid=$! 2>&1
+		log "apt-get upgrade" & pid=$!
 		wait
 		echo -e "\n* Installing packages... ${spin[0]}\n"
 		echo -e "\n* Going to suppress more messages that you don't need to worry about. Please wait... ${spin[0]}"
-		DEBIAN_FRONTEND=noninteractive apt-get -y install mariadb-server mariadb-client apache2 git unzip php5 php5-mcrypt php5-curl php5-mysql php5-gd php5-ldap libapache2-mod-php5 curl >> /var/log/snipeit-install.log & pid=$! 2>&1
+		log "DEBIAN_FRONTEND=noninteractive apt-get -y install mariadb-server mariadb-client apache2 git unzip php5 php5-mcrypt php5-curl php5-mysql php5-gd php5-ldap libapache2-mod-php5 curl" & pid=$!
 		progress
 		wait
 		echo -e "\n* Cloning Snipeit, extracting to $webdir/$name..."
-		git clone https://github.com/snipe/snipe-it $webdir/$name >> /var/log/snipeit-install.log & pid=$! 2>&1
+		log "git clone https://github.com/snipe/snipe-it $webdir/$name" & pid=$!
 		progress
-		php5enmod mcrypt >> /var/log/snipeit-install.log 2>&1
-		a2enmod rewrite >> /var/log/snipeit-install.log 2>&1
+		log "php5enmod mcrypt"
+		log "a2enmod rewrite"
 		vhenvfile
 		wait
 		echo >> $hosts "127.0.0.1 $hostname $fqdn"
-		a2ensite $name.conf
+		log "a2ensite $name.conf"
 		echo -e "* Modify the Snipe-It files necessary for a production environment.\n* Securing Mysql"
 		# Have user set own root password when securing install
 		# and just set the snipeit database user at the beginning
+		service mysql status >/dev/null || service mysql start
 		/usr/bin/mysql_secure_installation
 		echo -e "* Creating Mysql Database and User.\n##  Please Input your MySQL/MariaDB root password: "
 		mysql -u root -p < $dbsetup
 		cd $webdir/$name/
 		curl -sS https://getcomposer.org/installer | php
 		php composer.phar install --no-dev --prefer-source
 		perms
+		chown -R www-data:www-data "/var/www/$name"
```

Comment:
Was inside https://github.com/snipe/snipe-it/blob/master/snipeit.sh#L94 before, but i've re-used perms() for all OS'es.